projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -90,13 +95,37 @@ export class TuiInputPhoneInternational extends TuiControl<string> {
     @ViewChild(MaskitoDirective, {read: ElementRef})
     private readonly input?: ElementRef<HTMLInputElement>;
 
+    @ViewChildren(TuiOption, {read: ElementRef})
+    private readonly listOptions?: QueryList<ElementRef<HTMLButtonElement>>;
+
     protected readonly dropdown = tuiDropdown(null);
     protected readonly options = inject(TUI_INPUT_PHONE_INTERNATIONAL_OPTIONS);
     protected readonly size = inject(TUI_TEXTFIELD_OPTIONS).size;
     protected readonly open = tuiDropdownOpen();
     protected readonly names = toSignal(inject(TUI_COUNTRIES));
     protected readonly metadata = toSignal(from(this.options.metadata));
     protected readonly countryIsoCode = signal(this.options.countryIsoCode);
+    protected readonly icons = inject(TUI_COMMON_ICONS);
+    protected readonly internationalSearchLabel$ = inject(TUI_INTERNATIONAL_SEARCH);
+    protected readonly search = signal<string>('');
+
+    protected readonly filtered = computed(() =>
```

Comment:
Why do we now have 2 filtered signals? `countriesFilteredSignal` is just filtering countries once again.